src/sql/workbench/browser/editor/profiler/profilerInput.ts

```diff
@@ -126,14 +130,40 @@ export class ProfilerInput extends EditorInput implements IProfilerSession {
 	}
 
 	public override getName(): string {
```

Comment:
Can you also test this while trying to open XEL file? XEL file doesn't have a connection, hence no connection information is present.

Reply:
Will attempt to see what happens when opening XEL file, and will add the file name if possible.